### PR TITLE
[simple_kit]: Add MenuActionSheet

### PR DESCRIPTION
**Pull request by eli1stark**

### CODE REVIEW

File: plugins/simple_kit/lib/src/action_sheet/components/basic_bottom_sheet/basic_bottom_sheet.dart [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+void showBasicBottomSheet({
+  Widget? pinned,
+  Function(dynamic)? then,
+  Function()? onDissmis,
+  String? header,
+  double? maxHeight,
+  double? minHeight,
+  double? horizontalPadding,
+  bool scrollable = false,
+  Color color = Colors.white,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/action_sheet/components/basic_bottom_sheet/basic_bottom_sheet.dart:15]: not simple color

File: plugins/simple_kit/lib/src/action_sheet/components/basic_bottom_sheet/components/bottom_sheet_bar.dart [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    return Padding(
+      padding: EdgeInsets.only(
+        top: 10.h,
+        bottom: 15.h,
+      ),
+      child: Container(
+        width: 130.w,
+        height: 4.h,
+        decoration: BoxDecoration(
+          color: Colors.grey[300],
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/action_sheet/components/basic_bottom_sheet/components/bottom_sheet_bar.dart:20]: not simple color

File: plugins/simple_kit/lib/src/icons/24x24/base/simple_base_svg_w24x24.dart [hunk 3]
```diff
@@ -16,10 +16,10 @@ class SimpleBaseSvg20X20 extends StatelessWidget {
   Widget build(BuildContext context) {
     return ConstrainedBox(
       constraints: BoxConstraints(
-        maxWidth: 20.r,
-        maxHeight: 20.r,
-        minWidth: 20.r,
-        minHeight: 20.r,
+        maxWidth: 24.w,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/icons/24x24/base/simple_base_svg_w24x24.dart:19]: ??? :)

File: plugins/simple_kit/lib/src/action_sheet/components/basic_bottom_sheet/basic_bottom_sheet.dart (hunk 1 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/action_sheet/components/basic_bottom_sheet/basic_bottom_sheet.dart]: done

File: plugins/simple_kit/lib/src/action_sheet/components/basic_bottom_sheet/components/bottom_sheet_bar.dart (hunk 2 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/action_sheet/components/basic_bottom_sheet/components/bottom_sheet_bar.dart]: done

File: plugins/simple_kit/lib/src/icons/24x24/base/simple_base_svg_w24x24.dart (hunk 3 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/icons/24x24/base/simple_base_svg_w24x24.dart:21]: // Sometimes we need an icon to target height and sometimes width in order
// to satisfy design guides. This is why there exists BaseSvgR24 and BaseSvgW24

**Review by VovaMedvediev — APPROVED** [review]

**Pull request merged**